index.html

```diff
@@ -201,5 +201,19 @@ <h2 class="aheading">About Myself</h2>
   </div>
         </div>
       </section>
+      <div >
+      <footer class="contact-form"> 
+      <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] I think the form is not correctly integrated with the Formspree service hence giving back the following message when submitting the form.

**Output**
<img src="https://user-images.githubusercontent.com/17243325/207518822-1359f3e0-71de-42f2-a316-ebe7f0540248.png" width="300">

- [ ] Kindly visit the [Formspree](https://formspree.io/html/) website and get your own unique id to use for this form. You can replace `{form_id}` with your own unique id so the form data will be collected by the service and get the contents of the form in your email. 🤝️